content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -81,9 +85,105 @@ the path is relative to the KubeOne configuration file. This means that
 
 {{% notice note %}}
 Subdirectories are not considered when applying addons. Only addons in the root
-of the provided directory will be applied.
+of the provided directory will be applied. E.g. `./addons/addon_name1` is
+supported addon while `./addons/groupping/addon_name1` and
+`./addons/addon_name1/subdir1` does not.
+{{% /notice %}}
+
+## Embedded Addons
+
+Since v1.3 release KubeOne has new kind of addons that always caried with the
```

Comment:
I'd shuffle words a little bit. Also, some typo and grammar fixes.

```suggestion
Since KubeOne v1.3 release, there's a new kind of addons that are always carried with the
```